tox.ini

```diff
@@ -1,30 +1,28 @@
 [tox]
 downloadcache = {toxworkdir}/cache/
```

Comment:
downloadcache is entirely deprecated and has no effect.  I propose we remove

Reply:
Removed.